pom.xml

```diff
@@ -272,6 +273,14 @@
 				<groupId>com.google.cloud.tools</groupId>
 				<artifactId>jib-maven-plugin</artifactId>
 				<version>${jib-maven-plugin.version}</version>
+				<configuration>
+					<from>
+						<image>416670754337.dkr.ecr.eu-west-2.amazonaws.com/ci-corretto-build-21:latest</image>
+					</from>
+					<to>
+						<image>169942020521.dkr.ecr.eu-west-1.amazonaws.com/local/company-exemptions-data-api:latest</image>
```

Comment:
Once the pipeline is moved to shared services could you please make a note to update this to image reference from shared services ?

Reply:
comment added